docs/content/documentation/getting-started/installation.md

```diff
@@ -40,7 +40,7 @@ $ xbps-install -S gutenberg
 
 ## From source
 To build it from source, you will need to have Git, [Rust and Cargo](https://www.rust-lang.org/)
-installed.
+installed (also few compiling tools & libraries : make, libsass, g++, libssl-dev)
```

Comment:
which dependency requires `libssl-dev`?

Reply:
Also those are only the dependencies for OSX/Linus, Windows is a bit different. Maybe I should just point to the libsass installation guide

Reply:
True, I don't know for libssl, but from alpine image, it's not working without libssl (I've not checked which one was failing :/)